src/styles/blocks/catalog.scss

```diff
@@ -18,8 +18,16 @@
     border-radius: 5px;
     margin-bottom: 48px;
 
+    transition-duration: $transition-duration;
+    transition-timing-function: smooth;
```

Comment:
```suggestion
    transition: transform $transition-duration smooth;
```

Reply:
Better to specify transition propery. It impact on performance. By default it is all.
Please, check in second place it